src/ImageGallery.jsx

```diff
@@ -43,6 +43,7 @@ export default class ImageGallery extends React.Component {
     items: PropTypes.array.isRequired,
     showNav: PropTypes.bool,
     autoPlay: PropTypes.bool,
+  	bulletProps: PropTypes.object,
```

Comment:
is the spacing messed up here?

Reply:
Not seeing it on my machine. Git diff is showing it though.

```
 static propTypes = {
    flickThreshold: PropTypes.number,
    items: PropTypes.array.isRequired,
    showNav: PropTypes.bool,
    autoPlay: PropTypes.bool,
  	bulletProps: PropTypes.object,
    lazyLoad: PropTypes.bool,
    infinite: PropTypes.bool,
    showIndex: PropTypes.bool,
    showBullets: PropTypes.bool,
    showThumbnails: PropTypes.bool,
    showPlayButton: PropTypes.bool,
    showFullscreenButton: PropTypes.bool,
    disableThumbnailScroll: PropTypes.bool,
    disableArrowKeys: PropTypes.bool,
```